### ISSUE TITLE
Change volume

### ISSUE THREAD

**Issue by wissehes** [ChristopheJacquet/PiFmRds#104]
Hi, sorry if this isn't the right place to ask this, but is there a feature to change the volume of this command:
`sudo arecord -fS16_LE -r 44100 -Dplughw:1,0 -c 2 -  | sudo ./pi_fm_rds -audio -`
because the volume of the sound is very low...

thanks

**Comment by Marrley** [comment]
Have You Fixed It? Im Having The Same Problem

**Comment by TheChicken14** [comment]
No, I couldn't find a fix, but it's not a big issue if you just turn the volume on your radio up I guess

**Comment by KubaPro010** [comment]
you could use my fork of rpitx (and it does have pifmrds, with better rds encoder and better control over transmission,but for now the code is made for the pi 3 but can be very easy changed to a another rpi), you can specify a `-audiogain` arg or control the gain with the fifo (send to the fifo: `GAI {gain level}`, also gain is just how many times is the audio louder)
